cmd/expiration-mailer/main.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
+// How much earlier (than configured nag intervals) to send reminders,
+// to account for the expected delay before the next expiration-mailer
+// invocation.
+const nagMargin = 24 * time.Hour
```

Comment:
Please add a parameter like `MailerRunInterval` to `Mailer struct` in `cmd/shell.go` so that this is configurable.